classes/PodsUI.php

```diff
@@ -4653,9 +4656,21 @@ public function pagination( $header = false ) {
 
 		if ( false !== $this->pagination ) {
 			if ( 1 < $total_pages ) {
+				$first_link    = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=1' );
+				$prev_link     = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=' . max( $this->page - 1, 1 ) );
+				$next_link     = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=' . min( $this->page + 1, $total_pages ) );
```

Comment:
* :warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 5 spaces
* :no_entry_sign: Inline IF statements are not allowed

<sub>hash: a5cd5fd24b603ca4ab57e0735b02ed3a</sub>